dashboard/test/integration/pardot_test.rb

```diff
@@ -0,0 +1,87 @@
+require 'test_helper'
+require 'cdo/contact_rollups'
+
+PEGASUS_TEST_DB_NAME = "pegasus_#{Rails.env}"
+COLUMN_NAME_TO_INDEX_MAP = { "roles": 0, "ages_taught": 1 }.freeze
+
+class PardotTest < ActiveSupport::TestCase
+  def test_empty_contacts
+    # Test the rollup process with an empty database
+    rollups_test_helper 0, []
+  end
+
+  def test_teachers
```

Comment:
Because the rollup process operates on a large corpus of data, I am going to leave this as is. If I split it up, each test would be operating on a database with a small number of users (such as one). By generating a mildly interesting body of test fixtures, it demonstrates that we don't have something going on such as bad joins that will generate additional rows in the output. Things like that might not show up in the smaller data set of the individual things we are verifying later.